src/rgw/rgw_rest_pubsub.cc

```diff
@@ -1071,6 +1099,128 @@ int RGWPSCreateNotifOp::verify_permission(optional_yield y) {
   return 0;
 }
 
+void RGWPSCreateNotifOp::execute_v2(optional_yield y) {
+  op_ret = verify_params();
```

Comment:
lines 1103 - 1112 are common to `execute()` and `execute_v2()`

Reply:
> lines 1103 - 1112 are common to `execute()` and `execute_v2()`

I purposely have the 2 functions separate, for readability n also in future when we remove v1, we can directly  remove execute and rename execute-v2.